packages/vscode-extension/package.nls.json

```diff
@@ -112,7 +112,7 @@
     "teamstoolkit.commandsTreeViewProvider.createProjectTitle": "Create a New App",
     "teamstoolkit.commandsTreeViewProvider.deploy.blockTooltip": "Cannot run command when deploying. Try again when deployment is finished.",
     "teamstoolkit.commandsTreeViewProvider.deploy.running": "Deploying to the cloud...",
-    "teamstoolkit.commandsTreeViewProvider.deployDescription": "Deploy your Teams app to the provisioned resources in Azure",
+    "teamstoolkit.commandsTreeViewProvider.deployDescription": "Execute the deploy lifecycle in teamsapp.yml",
```

Comment:
deploy lifecycle -> 'deploy' lifecycle stage

same for the rest lines.

Reply:
@sffamily Thanks for the comment, please help to review again.